### ISSUE TITLE
M6 is missing

### ISSUE THREAD

**Issue by BerkBorak** [julielerman/PluralsightEFCore2GettingStarted#1]
The Web Application created in Module 6 is missing. The code for the web application isn't shown as well so it's quite hard to follow without prior knowledge in Web Applications.

**Comment by julielerman** [comment]
Module # confusion. :) I create the modules as 1-5. Then Pluralsight adds a little overview in front. So #5 appears to be the 6th one on the website. So the web app is there. I'll rename the folders for clarity. Hope that helps. OK to close?

**Comment by BerkBorak** [comment]
Thanks for clearing it up, I should've checked more thoroughly.

**Comment by julielerman** [comment]
I'm glad you pointed it out because it got me to clarify the names of the folders etc. So thank you! :)
